src/index.html

```diff
@@ -1,13 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com"
+    >
+
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+    >
+
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet"
+    >
+
+    <link rel="stylesheet"
+          href="./styles/index.scss"
+    >
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card"
+         data-qa="card"
+    >
+      <img src="images/imac.jpeg"
+           alt="product item imac"
+           class="card__photo"
+      >
```

Comment:
```suggestion
      <img 
        src="images/imac.jpeg"
        alt="product item imac"
        class="card__photo"
      >
```

Reply:
Here is good example)
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/49da56f5-0e5d-4ede-b0e3-d142d4f2a25e)

Reply:
do it in all cases